‘uint32_t’ was not declared in this scope

The following bug was reported using the latest git commit:
https://github.com/pentoo/pentoo-overlay/issues/1519
```
In file included from be20_api/atomic_unicode_histogram.cpp:10:
be20_api/unicode_escape.h:29:26: error: ‘uint32_t’ was not declared in this scope; did you mean ‘utf8::uint32_t’?
   29 | bool valid_utf8codepoint(uint32_t unichar); // not all unichars are valid codepoints
      |                          ^~~~~~~~
      |                          utf8::uint32_t
In file included from be20_api/utf8/checked.h:31,
                 from be20_api/utf8.h:30,
                 from be20_api/unicode_escape.h:18:
be20_api/utf8/core.h:57:29: note: ‘utf8::uint32_t’ declared here
```
Please advise

--- a/unicode_escape.h
+++ b/unicode_escape.h
@@ -7,6 +7,7 @@
 #ifndef UNICODE_ESCAPE_H
 #define UNICODE_ESCAPE_H
 
+#include <cstdint>
 #include <cstring>
 #include <cwctype>
 #include <iostream>

